example/test_problem/Hydro/CCSN/Input__Flag_Lohner.CoreCollapse

```diff
@@ -0,0 +1,10 @@
+# Level    Threshold_Refine  Threshold_Derefine              Filter              Soften          MinDensity
+      0                0.80                0.40                0.01                0.00                0.00
+      1                0.80                0.40                0.01                0.00                0.00
+      2                0.80                0.40                0.01                0.00                0.00
+      3                0.80                0.40                0.01                0.00                0.00
+      4                0.80                0.40                0.01                0.00                0.00
+      5                0.80                0.40                0.01                0.00                0.00
+      6                0.80                0.40                0.01                0.00                0.00
+      7                0.80                0.40                0.01                0.00                0.00
+      8                0.80                0.40                0.01                0.00                0.00
```

Comment:
Why adopt `Threshold_Derefine = 0.4`?

Reply:
I chose this value from @hfhsieh 's test.
https://gamer-project.slack.com/archives/CCM5BFPQB/p1657688712392319

Reply:
That test is outdated.